recipe/conda_build_config.yaml

```diff
@@ -5,7 +5,7 @@ cxx_compiler:          # [win]
 MACOSX_SDK_VERSION:    # [osx and x86_64]
   - 10.12              # [osx and x86_64]
 channel_targets:
-  - conda-forge llvm_rc
+  - conda-forge llvm_dev
```

Comment:
I'd prefer a dedicated label (and branch...?) `llvm_tf`

Reply:
In the sense that it's not a dev-package (i.e. not meant for consumption). If people were to install a new tensorflow and get packages from `conda-forge/label/llvm_dev` that feels much more iffy than `conda-forge/label/llvm_dev`.

Reply:
These packages are only installed during the build. As we link statically, end-users will never see them